tests/e2e/common.go

```diff
@@ -478,14 +482,13 @@ func (td *OsmTestData) CreateMultipleNs(nsName ...string) error {
 // CreateNs creates a Namespace. Will automatically add Docker registry creds if provided
 func (td *OsmTestData) CreateNs(nsName string, labels map[string]string) error {
 	if labels == nil {
-		labels = map[string]string{}
+		labels = td.GetTestSelectorMap()
+	} else {
+		for k, v := range td.GetTestSelectorMap() {
+			labels[k] = v
+		}
```

Comment:
Could this loop be run unconditionally so we could get rid of the if/else?

Reply:
hmmm... yeah can do. I'll do.